src/Akeneo/Apps/back/Domain/Model/Write/App.php

```diff
@@ -80,4 +91,9 @@ public function setFlowType(FlowType $flowType): void
     {
         $this->flowType = $flowType;
     }
+
+    public function setImage(?AppImage $image): void
```

Comment:
According to a previous comment. If we want to remove the image, a method removeImage can be added :)
```suggestion
    public function setImage(AppImage $image): void
```

Reply:
Idem here :+1: